script/lib/utils.js

```diff
@@ -23,6 +23,17 @@ function getElectronExec () {
   }
 }
 
+function getOutDir () {
+  if (process.env.ELECTRON_OUT_DIR) {
+    return process.env.ELECTRON_OUT_DIR
+  } else {
+    for (const buildType of ['Debug', 'Testing', 'Release']) {
+      const outPath = path.resolve(SRC_DIR, 'out', buildType)
+      if (fs.existsSync(outPath)) return buildType
```

Comment:
We should add console output here so that folks see what dir was automatically used.

Reply:
hmm, could we perhaps do this conditionally? Since this will be used in the GN check lint step, i don't imagine folks would want the extra console spew in their terminals every time 🤔